### PR TITLE
'subfolder' issue fixed completely

### PR DESCRIPTION
**Pull request by jinseuk56**
Added the option to convert all MIB files in 'Merlin' folder without the need to specify 'subfolder'.
For example, if this option is activated, and 'subfolder' is not specified,
(1) MIB files in 'Merlin' folder with their subfolders will be saved in '../session/processing/Merlin/subfolder/'
(2) MIB files in 'Merlin' folder without subfolders will be saved in ../session/processing/Merlin/'

The 'subfolder' widget has a higher priority than the 'convert all MIB files' option.
In other words, if 'subfolder' is specified, the MIB files in '../session/Merlin/subfolder/' will be converted regardless of the 'convert all MIB files' option.
widgets

### CODE REVIEW

**Comment by M0hsend** [comment]
Thanks Jinseok!
